.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+  // See http://go.microsoft.com/fwlink/?LinkId=827846
+  // for the documentation about the extensions.json format
+  "recommendations": [
```

Comment:
What's this? I'm curious

Reply:
When this project folder is opened, VS Code will prompt the user to suggest that these extensions are installed when working on this project.